pkg/templates/ignition.go

```diff
@@ -61,6 +63,26 @@ func (i *ignition) GenerateNode(kluster *kubernikusv1.Kluster, secret *v1.Secret
 		return nil, err
 	}
 
+	//this is the old default for backwards comptibility with clusters that don't have a passwort generated
+	//TODO: Remove once all klusters are upgraded
+	passwordHash := "$6$rounds=1000000$aldshc,xbneroyw$I756LN/FtceE1deC2H.tGeSdeeelaaZWRwzmbEuO1SANf7ssyPjnbQjlW/FcMvWGUGrhF64tX9fK0abE/4oQ80"
```

Comment:
wth? 🤣

Reply:
Thats just the old password  salted and hashed 1 million times with sha512. Nothing special.
![image](https://user-images.githubusercontent.com/439223/34876469-fc78c416-f7a1-11e7-9c03-cec965a2598b.png)